tidb-limitations.md

```diff
@@ -0,0 +1,74 @@
+---
+title: TiDB Limitations
+summary: Learn the limitations of TiDB.
+category: introduction
+---
+
+# TiDB Limitations
+
+This document describes the common limitations of TiDB, including the maximum identifier length and the maximum count of supported databases, tables, indexes, partitioned tables, and sequences.
+
+## Limitation on identifier length
+
+| Identifier type | Maximum length (character) |
+|:---------|:--------------|
+| Database | 64 |
+| Table    | 64 |
+| Column   | 64 |
+| Index    | 64 |
+| View     | 64 |
+| Sequence | 64 |
+
+## Limitation on the total count of databases, tables, views, and connections
+
+| Identifier type  | Maximum count   |
+|:----------|:----------|
+| Databases | unlimited |
+| Tables    | unlimited |
+| Views     | unlimited |
+| Connections| unlimited|
+
+## Limitation on a single database
+
+| Type       | Upper limit   |
+|:----------|:----------|
+| Tables    | unlimited  |
+
+## Limitation on a single table
+
+| Type       | Upper limit   |
+|:----------|:----------|
+| Columns   | 512       |
+| Indexes    | 64        |
+| Rows      | unlimited |
+| Size      | unlimited |
+| Partitions | 1024      |
+
+## Limitation on a single row
+
+| Type       | Upper limit   |
+|:----------|:----------|
+| Size       | 6 MB       |
+
+## Limitation on a single column
+
+| Type       | Upper limit   |
+|:----------|:----------|
+| Size       | 6 MB       |
+
+## Limitation on string types
```

Comment:
ditto

Reply:
In this context, the plural form of `limitation` is more commonly used. Please pay attention to this word.